plugins/qbAclPlugin/lib/QubitAcl.class.php

```diff
@@ -73,6 +73,11 @@ public static function getInstance()
     return self::$_instance;
   }
 
+  public static function destruct()
```

Comment:
This method could use some docs to describe our very particular use case, perhaps something like:

```
Each request is completely unique in PHP however our workers run in a loop
where state is held across jobs. This class was built as a singleton so we
need this extra method to void the instance when needed.
```